erc20/alias-erc20-deploy.js

```diff
@@ -54,48 +54,48 @@ async function deploy(info) {
+  await deploy(conf.bridges[0].parent.url, conf.sender.parent, conf.contract.parent);
 
   // add minter
-  await conf.child.newInstance.methods.addMinter(conf.child.bridge).send({ from: conf.child.sender, to: conf.child.bridge, gas: 100000000, value: 0 });
-  await conf.parent.newInstance.methods.addMinter(conf.parent.bridge).send({ from: conf.parent.sender, to: conf.child.bridge, gas: 100000000, value: 0 });
-
-  // register operator
-  await conf.child.newInstanceBridge.methods.registerOperator(conf.child.operator).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.registerOperator(conf.parent.operator).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+  await conf.contract.child.newInstance.methods.addMinter(conf.contract.child.bridge).send({ from: conf.sender.child.address, to: conf.contract.child.bridge, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstance.methods.addMinter(conf.contract.parent.bridge).send({ from: conf.sender.parent.address, to: conf.contract.parent.bridge, gas: 100000000, value: 0 });
 
   // register token
-  await conf.child.newInstanceBridge.methods.registerToken(conf.child.token, conf.parent.token).send({ from: conf.child.sender, gas: 100000000, value: 0 });
-  await conf.parent.newInstanceBridge.methods.registerToken(conf.parent.token, conf.child.token).send({ from: conf.parent.sender, gas: 100000000, value: 0 });
+  await conf.contract.child.newInstanceBridge.methods.registerToken(conf.contract.child.token, conf.contract.parent.token).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+  await conf.contract.parent.newInstanceBridge.methods.registerToken(conf.contract.parent.token, conf.contract.child.token).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+  for (const bridge of conf.bridges) {
+    // register operator
+    await conf.child.newInstanceBridge.methods.registerOperator(bridge.child.operator).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
+    await conf.parent.newInstanceBridge.methods.registerOperator(bridge.parent.operator).send({ from: conf.sender.parent.address, gas: 100000000, value: 0 });
+
+    const alias = "MYBRIDGE2"
+    const url = bridge.child.url;
+    log = 'registering bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_registerBridgeByAlias', [alias, conf.contract.child.bridge, conf.contract.parent.bridge]);
+
+    log = 'subscribing bridges to the child node'
+    await jsonRpcReq(url, log, 'subbridge_subscribeBridgeByAlias', [alias]);
+
+    log = 'register token to subbridge..'
+    await jsonRpcReq(url, log, 'subbridge_registerTokenByAlias', [alias, conf.contract.child.token, conf.contract.parent.token]);
+  }
+
+  // setOperatorThreshold
+  await conf.contract.child.newInstanceBridge.methods.setOperatorThreshold(conf.bridges.length).send({ from: conf.sender.child.address, gas: 100000000, value: 0 });
```

Comment:
This method enables examples to work in the HA setting. A brief description seems needed what it is for and why it is necessary.

Reply:
@hyunsooda Setting the threshold to the length of bridges is NOT necessary, however I thought it would be helpful to find out if all bridges are working fine - if one of the bridges is malfunctioning, the value transfer won't succeed.
But I agree that I should add clarification on that, thank you.

Reply:
ok good thx.